zerobuf/render/lookOut.fbs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori.Chevtchenko@epfl.ch
+//
+namespace zerobuf.render;
+
+table LookOut
```

Comment:
Add description about what transformation this LookOut defines, matrix layout, unit, ...